hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,6 @@
+RELEASE_TYPE: patch
+
+This patch fixes the ``is_hypothesis_file`` internal function.
```

Comment:
We try to keep our changelogs only describing public API, so I'd say e.g.

"This patch fixes a bug where errors in third-party extensions such as :pypi:`hypothesis-trio` or  :pypi:`hypothesis-jsonschema` were incorrectly considered to be Hypothesis internal errors, which could result in confusing error messages.

Thanks to Vincent Michel for reporting and fixing the bug!"

(and I should have mentioned this last time, but please add yourself to the authors list at the bottom on CONTRIBUTING.rst!)

Reply:
Done!